python/lsst/pipe/supertask/activator.py

```diff
@@ -131,6 +132,34 @@ def activate(self):
         """
         pass
 
+def get_inheritance(stask_class, subclasses=[]):
```

Comment:
Its recursive, so the list needs to be declared and passed recursively.  I can change it for,
`def get_inheritance(stask_class, subclasses = None)
    if subclasses is None: subclasses = []`
??